Formula/astrometry-net.rb

```diff
@@ -0,0 +1,66 @@
+class AstrometryNet < Formula
+  include Language::Python::Virtualenv
+
+  desc "Automatic identification of astronomical images"
+  homepage "https://github.com/dstndstn/astrometry.net"
+  url "https://github.com/dstndstn/astrometry.net/releases/download/0.76/astrometry.net-0.76.tar.gz"
+  sha256 "244355ef9716a9e062eb19e8547e8b50434c7df52a8b96f9d0a254bc646d1f0d"
+
+  depends_on "pkg-config" => :build
+  depends_on "swig" => :build
+  depends_on "cairo"
+  depends_on "cfitsio"
+  depends_on "gsl"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "netpbm"
+  depends_on "numpy"
+  depends_on "python"
+  depends_on "wcslib"
+
+  resource "numpy" do
+    url "https://files.pythonhosted.org/packages/83/6b/d03277eacf113697675cd659086a4dcf9472108e2f1a83884c0271bdca46/numpy-1.15.3.zip"
+    sha256 "1c0c80e74759fa4942298044274f2c11b08c86230b25b8b819e55e644f5ff2b6"
+  end
```

Comment:
You have both `depends_on "numpy"` and this `numpy` resource. Can this resource be removed?

Reply:
Good catch.

Reply:
We're using virtualenv to vendor a package (fitsio) that depends on numpy.  If we have 'numpy' as a dependency, does that package become available within the virtualenv?  Or does it have to get added to the virtualenv?  (Or do we have to use "virtualenv --system-site-packages"?)

Reply:
Yes, it looks like `virtualenv --system-site-packages` would you what you want.

Rather than using `virtualenv` to install a resource, another option is to use `pip3` to install `fits`.
```ruby
xy = Language::Python.major_minor_version "python3"
ENV.prepend_path "PYTHONPATH", libexec/"lib/python#{xy}/site-packages"
system "pip3", "install", "--prefix=#{libexec}", "fitsio=0.9.11"
…
bin.env_script_all_files libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"]
```

Reply:
I would love that pip3 solution; way easier, in my opinion.

Does the env_script_all_files magic edit scripts to modify the pythonpath at the beginning of the scripts in the bin directory?  (assuming shell scripts?  Some of the scripts in bin/ (not all ending in .py) are python scripts starting with "#! /usr/bin/env python3" or the equivalent.

But isn't this pip solution somewhat contrary to the "Python for Formula Authors" advice to install 'applications' in a virtualenv?

Reply:
> But isn't this pip solution somewhat contrary to the "Python for Formula Authors" advice to install 'applications' in a virtualenv?

I'll defer to a Homebrew/core maintainer that has more experience than myself. I've used the above `pip3` snippet a fair bit in Brewsci/bio.

Reply:
@commitay, @javian, @fxcoudert or anyone else from @Homebrew/core with more python package experience than me, could you please weigh in here when you have a chance? Thanks.

Reply:
> Does the env_script_all_files magic edit scripts to modify the pythonpath at the beginning of the scripts in the bin directory?

It moves all executables found in `bin` to `libexec/bin`, then creates a shell script in `bin` that sets `PYTHONPATH`  and calls the real executable that has been moved to `libexec/bin`.

Reply:
Oh, cool, that's handy.

Thanks for your comments and advice here!